src/vs/workbench/parts/debug/electron-browser/replViewer.ts

```diff
@@ -7,13 +7,10 @@ import * as nls from 'vs/nls';
 import { TPromise } from 'vs/base/common/winjs.base';
```

Comment:
Looks good overall and I like we removed code from this class.

I only have the issue that you are creating a `linkDetector` at every iteration in a loop. Would it not make sense just to create one global one for the replViewer and always to reuse it?